charts/pg-operator/templates/NOTES.txt

```diff
@@ -12,4 +12,13 @@
 
     helm install my-db percona/pg-db --namespace={{ .Release.Namespace }}
 
+{{- if .Release.IsUpgrade }}
+
+** WARNING ** During Helm upgrade CRDs are not automatically upgraded.
+
+Please manually apply any new CRDs:
```

Comment:
Unless I am missing smth, maybe it'll be better to add a link to our official docs and provide here the same commands? WDYT
 https://docs.percona.com/percona-operator-for-postgresql/2.0/update.html#upgrade-via-helm ? 
 `$ kubectl apply --server-side --force-conflicts -f https://raw.githubusercontent.com/percona/percona-postgresql-operator/v2.7.0/deploy/crd.yaml
$ kubectl apply -f https://raw.githubusercontent.com/percona/percona-postgresql-operator/v2.7.0/deploy/rbac.yaml -n postgres-operator`
But we will need to update it for every release.

Reply:
No need to change every release, just need to use the {{ .Chart.AppVersion }} with the updated version :)

Also I don't think rbac should be part of the command, since it's possible to apply updates to the roles with helm upgrade.

WDYT about the namespace?

Reply:
Actually crd doesn't really namespace, only rbac, just ignore the question